fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -836,6 +840,14 @@ showProcStatement
     : SHOW PROC path=string
     ;
 
+// ---------------------------------------- Backup Store Statement -----------------------------------------------------
+createRepositoryStatement
+    : CREATE (READ ONLY)? REPOSITORY identifier
+    WITH BROKER identifier
+    ON 'LOCATION' string
```

Comment:
Move 'LOCATION' to StarRocksLex.g4, and add it in nonReserved section

Reply:
It has been amended